src/smc-webapp/billing.cjsx

```diff
@@ -1863,7 +1865,7 @@ BillingPage = rclass
                 </div>
             else if @props.is_simplified
```

Comment:
There is unnecessary code duplication here. It should just be `else` and then the piece below `<Subscriptions ...` can be wrapped in an `if` block.